apache-httpd.conf

```diff
@@ -0,0 +1,544 @@
+#
```

Comment:
Please remove all the comments from this file - we don't need 550 lines of comments in the repo.